rotkehlchen/chain/evm/decoding/beefy_finance/utils.py

```diff
@@ -201,6 +201,21 @@ def _query_beefy_cow_token_price(
     return Price(usd_price)
 
 
+def _get_underlying_token(vault_token: 'EvmToken') -> 'EvmToken | None':
+    """Gets the underlying token for a Beefy vault token."""
```

Comment:
And if you do rename to get_single_underlying_token and adjust docstring

```suggestion
    """adjust me"""
```